parser-typechecker/src/Unison/Lexer.hs

```diff
@@ -236,7 +236,14 @@ lexer0 scope rem =
       (spaces, '-':'-':rem) -> spanThru' (/= '\n') rem $ \(ignored, rem) ->
         pushLayout b l (incBy ('-':'-':ignored) . incBy spaces $ pos) rem
       (spaces, rem) ->
-        let pos' = incBy spaces pos in go ((b, column pos') : l) pos' rem
+        let topcol = top l
+            pos' = incBy spaces pos
+            col' = column pos'
+        in
+          if b == "=" && col' <= topcol then
+            popLayout0 ((b, col' + 1) : l) pos' rem
```

Comment:
the +1 here is a bit dirty but it forces a close on popLayout0

Reply:
Ah interesting, thanks for pointing this out, but could you include this as a comment in the actual source?

Reply:
@pchiusano ok like this?